### PR TITLE
Show the time units only in the hint message

### PR DESCRIPTION
**Pull request by akopich**
Also, DisplayFormats.h/cpp is refactored

Tests: Unit and Manual
Bug: http://b/224731226

### CODE REVIEW

File: src/DisplayFormats/DisplayFormats.cpp [hunk 1]
```diff
@@ -22,27 +24,77 @@ std::string GetDisplaySize(uint64_t size_bytes) {
...
+      return absl::ToDoubleHours(duration) / kHoursInOneDay;
+    default:
+      ORBIT_UNREACHABLE();
+  }
+}
+
+std::string GetDisplayTime(absl::Duration duration) {
+  const TimeUnit unit = ChooseUnitForDisplayTime(duration);
+  const std::string unit_str = DisplayTimeUnit(unit);
+  const double duration_double = ToDoubleTimeUnits(duration, unit);
```
Comment (beckerhe) [inline src/DisplayFormats/DisplayFormats.cpp:96]: Nit/Personal preference: I'm not sure if having these two temporary variables makes it more readable.

File: src/DisplayFormats/include/DisplayFormats/DisplayFormats.h [hunk 2]
```diff
@@ -11,6 +11,10 @@ namespace orbit_display_formats {
 
 [[nodiscard]] std::string GetDisplaySize(uint64_t size_bytes);
 
+enum class TimeUnit { kNanosecond, kMicrosecod, kMillisecond, kSecond, kMinute, kHour, kDay };
+std::string DisplayTimeUnit(TimeUnit unit);
+TimeUnit ChooseUnitForDisplayTime(absl::Duration duration);
+double ToDoubleTimeUnits(absl::Duration duration, TimeUnit unit);
```
Comment (beckerhe) [inline src/DisplayFormats/include/DisplayFormats/DisplayFormats.h:17]: nodiscard

File: src/DisplayFormats/include/DisplayFormats/DisplayFormats.h [hunk 3]
```diff
@@ -11,6 +11,10 @@ namespace orbit_display_formats {
 
 [[nodiscard]] std::string GetDisplaySize(uint64_t size_bytes);
 
+enum class TimeUnit { kNanosecond, kMicrosecod, kMillisecond, kSecond, kMinute, kHour, kDay };
+std::string DisplayTimeUnit(TimeUnit unit);
```
Comment (beckerhe) [inline src/DisplayFormats/include/DisplayFormats/DisplayFormats.h:15]: I would suggest to call this function `GetDisplayTimeUnit` which makes it consistent with the other functions below.

File: src/DisplayFormats/include/DisplayFormats/DisplayFormats.h (hunk 2 above)
Comment (beckerhe) [inline src/DisplayFormats/include/DisplayFormats/DisplayFormats.h:17]: I assume this function is named like the abseil functions `ToDoubleNanoseconds`, right? While for the abseil functions with the given unit it's clear to me what it means, it's not clear to me what `ToDoubleTimeUnits` when I see this somewhere in the code. I'm wondering what could be a clearer name. Maybe `ToDoubleInGivenTimeUnit`? (I don't feel strongly about this. If we don't find a better name, feel free to leave it.)

**Review by beckerhe — APPROVED** [review]
Otherwise LGTM

File: src/DisplayFormats/DisplayFormats.cpp (hunk 1 above)
Comment (akopich) [reply src/DisplayFormats/DisplayFormats.cpp:96]: Fair enough.

**Review by florian-kuebler — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**